Showcase/ShowcaseSnapshotTests/ShowcaseSnapshotSwiftUITests.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
```

Comment:
On files in Sources, we wanted to have a header with the name of the project and the license. I don't know if we also want them here 🤔

Reply:
Ok I will fix it thanks for the review.